### PR TITLE
Create yarn-build-frontend.yml

### PR DESCRIPTION
**Pull request by akroon3r**
# Description

This PR includes the following proposed change(s):

None.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Nope.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by akroon3r** [comment]
Treating warnings as errors because process.env.CI = true.
Most CI servers set it automatically.

Failed to compile.

./src/components/customers/Customers.js
  Line 25:9:   'userIcon' is assigned a value but never used                                                                  no-unused-vars
  Line 29:6:   React Hook useEffect has a missing dependency: 'customerId'. Either include it or remove the dependency array  react-hooks/exhaustive-deps
  Line 40:13:  img elements must have an alt prop, either with meaningful text, or an empty string for decorative images      jsx-a11y/alt-text

**Comment by akroon3r** [comment]
Treating warnings as errors because process.env.CI = true.
Most CI servers set it automatically.

Failed to compile.

  Line 29:6:   React Hook useEffect has a missing dependency: 'customerId'. Either include it or remove the dependency array  react-hooks/exhaustive-deps

**Comment by akroon3r** [comment]
Treating warnings as errors because process.env.CI = true.
Most CI servers set it automatically.

Failed to compile.

  Line 40:13:  img elements must have an alt prop, either with meaningful text, or an empty string for decorative images      jsx-a11y/alt-text

**Comment by akroon3r** [comment]
@sdevalapurkar could you look at these

**Review by sdevalapurkar — APPROVED** [review]
It is fixed now :) thank you

File: .github/workflows/yarn-build-frontend.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    name: Make sure frontend installs and builds
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      # Run Yarn Install - Oracle Modernization Frontend
+      - name: Run Yarn Install - Oracle Modernization Frontend
+        run: yarn --cwd src/frontend install
+        
+      # Run Yarn Install - Oracle Modernization Frontend
```
Comment (sdevalapurkar) [inline .github/workflows/yarn-build-frontend.yml:19]:
```suggestion
      # Run Yarn Build- Oracle Modernization Frontend
```

File: .github/workflows/yarn-build-frontend.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+      - name: Run Yarn Install - Oracle Modernization Frontend
```
Comment (sdevalapurkar) [inline .github/workflows/yarn-build-frontend.yml:20]:
```suggestion
      - name: Run Yarn Build - Oracle Modernization Frontend
```

**Review by sdevalapurkar — CHANGES_REQUESTED** [review]

File: .github/workflows/yarn-build-frontend.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    name: Make sure frontend installs and builds
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      # Run Yarn Install - Oracle Modernization Frontend
+      - name: Run Yarn Install - Oracle Modernization Frontend
+        run: yarn --cwd src/frontend install
+        
+      # Run Yarn Build- Oracle Modernization Frontend
```
Comment (sdevalapurkar) [inline .github/workflows/yarn-build-frontend.yml:19]:
```suggestion
      # Run Yarn Build - Oracle Modernization Frontend
```

**Review by sdevalapurkar — CHANGES_REQUESTED** [review]

**Review by sdevalapurkar — APPROVED** [review]

**Pull request merged**